bestpractices/content/passwords.md

```diff
@@ -0,0 +1,64 @@
+# Password Management
+
+Maintaining open source software often means juggling a variety of responsibilities—from ensuring code quality to building a thriving community of contributors. Projects often find themselves juggling accounts on a variety of platforms like code repositories domain providers, and container repositories, all while trying to control access to the right set of individuals. This is often messy and a source of vulnerability.
+
+This post aims to help open source maintainers understand:
+
+1. The value of using a password manager for both personal and project credentials.  
+2. How to scan for accidentally committed secrets using open source tools like [trufflehog](https://github.com/trufflesecurity/trufflehog).  
+3. Techniques for setting up pre-commit hooks and CI scans to ensure secrets do not leak into your repositories.
+
+---
+
+## Risks of Insecure Password Management
+
+Hardcoded passwords, API keys, and other secrets in your repository can expose your project to risks like:
+
+- **Account Takeover**: Exposed credentials can allow unauthorized access to critical services (e.g., package registries, CI/CD systems, or infrastructure).  
+- **Reputation Damage**: A security breach can harm your project's reputation, deterring potential collaborators or users.
+- **User Supply Chain Risk**: With unauthorized access, attackers can push harmful content to your users. Most often, this is simply defacement or malicious binaries (like cryptojackers), but can sometimes be more insidious.
+
+Ensuring that passwords and other sensitive credentials remain secure is essential for any project that wants to maintain trust and protect both its developers and users.
+
+---
+
+## Best Practices for Secret Handling
+
+1. **Never store passwords or tokens directly in your repository.** Use something like GitHub Secrets to manage your secrets during CI/CD.
```

Comment:
```suggestion
1. **Never store passwords or tokens directly in your repository.** Use secrets on GitHub or similar tooling to manage your secrets during CI/CD.
```

"GitHub Secrets" makes me feel as if it's an official product/service offered by GitHub to manage secrets, and it doesn't show up as such on search engines, so it could be misleading – I'm curious to hear your thoughts.

Reply:
Good point, I can clarify with "[secrets in GitHub Actions](https://docs.github.com/en/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions)"